src/main/java/pl/zaczarowaneslodkosci/service/CakeServiceImpl.java

```diff
@@ -0,0 +1,48 @@
+package pl.zaczarowaneslodkosci.service;
+
+import org.springframework.stereotype.Service;
+import pl.zaczarowaneslodkosci.dto.CakeDtoRequest;
+import pl.zaczarowaneslodkosci.dto.CakeDtoResponse;
+import pl.zaczarowaneslodkosci.entity.CakeEntity;
+import pl.zaczarowaneslodkosci.repository.CakeRepository;
+
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+import static org.springframework.beans.BeanUtils.copyProperties;
+
+@Service
+public class CakeServiceImpl implements CakeService {
```

Comment:
tutaj brakuje arcy-ważnej adnotacji, @Transactional - poczytaj co to robi i po co to MUST HAVE!!!

Reply:
resolved